Formula/coq.rb

```diff
@@ -1,8 +1,7 @@
 class Coq < Formula
   desc "Proof assistant for higher-order logic"
   homepage "https://coq.inria.fr/"
-  url "https://github.com/coq/coq/archive/V8.12.2.tar.gz"
-  sha256 "2c57416e3ec737b212610512eae7e40259fb17a4e487b49981556f28838e8b17"
```

Comment:
You need to add back the `sha256` line and update it accordingly.

Reply:
Sorry, these were for testing and not intended to be part of this PR.